### DISCUSSION TITLE
download GRBL and Custom Buttons

### DISCUSSION THREAD

**Discussion by bruce1369** [arkypita/LaserGRBL#1289] [Q&A]
When I try to download GRBL my computer says it is a threat to my computer. Also cannot download Custom Buttons.
A clear and concise description of what the bug is.  Computer claims GRBL is a threat to computer. I have a new HP with windows 10 64 bit. Already ruined one old and another new computer When I did download GRBL.

**Comment by arkypita** [comment]
Typing the same request multiple times won't decrease the time you get a response, it's just annoying.

The reason windows warn you that the program can harm your computer is explained here http://lasergrbl.com/faq/ and even better here http://lasergrbl.com/does-lasergrbl-contain-viruses/.

Similarly, the procedure for installing the custom buttons is explained here https://lasergrbl.com/usage/custom-buttons/
You don't need to unzip the downloaded file, just follow the procedure.
